### DISCUSSION TITLE
cancer population parameter

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1356] [Questions & Problems]
Hello everyone, my project needs to establish PBPK model of cancer population, there is no default cancer population in pk-sim, I have been trying to find parameter adjustment of cancer population modeling, so far I have not found many articles, would you like to ask if you have experience in this field? Looking forward to your reply!

**Comment by prvmalik** [comment]
Cancer population tends to be leaner, older patients. More variable or lower plasma protein binding. Blood cancers may affect hematocrit. Liver cancers may affect hepatic metabolic capacity. Kidney cancers may affect GFR. In general there is a state of inflammation that affects disposition and/or catabolism of proteins. CYP3A4 metabolism may be mildly inhibited along with other enzymes.

**Reply by WTY-tt** [reply]
Thank you for your answer. The literature I have found so far is basically about CYP3A4, gastric emptying time, hematocrit and other parameters.

**Comment by WTY-tt** [comment]
Thank you for your answer. The literature I have found so far is basically about CYP3A4, gastric emptying time, hematocrit and other parameters.
